index.html

```diff
@@ -1,51 +1,54 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Agency</title>
+    <title>Laverage</title>
     <link rel="stylesheet" href="./css/main.css">
 </head>
+
 <body>
     <!-- header: start -->
     <header id="main_header"></header>
     <!-- header: end -->
     <!-- hero: start -->
     <div id="hero"></div>
     <!-- hero: end -->
-    <!-- services: start -->
-    <div id="services"></div>
-    <!-- services: end -->
+    <!-- features: start -->
+    <div id="features"></div>
+    <!-- features: end -->
     <!-- about_us: start -->
     <div id="about_us"></div>
     <!-- about_us: end -->
+    <!-- main-video: start -->
+    <div id="main-video"></div>
+    <!-- main-video: end -->
+    <!-- skills: start -->
+    <div id="skills"></div>
+    <!-- skills: end -->
+    <!-- services: start -->
+    <div id="services"></div>
+    <!-- services: end -->
     <!-- portfolio: start -->
     <div id="portfolio"></div>
     <!-- portfolio: end -->
-    <!-- fun_facts: start -->
-    <div id="fun_facts"></div>
-    <!-- fun_facts: end -->
-    <!-- team: start -->
-    <div id="team"></div>
-    <!-- team: end -->
     <!-- testimonials: start -->
     <div id="testimonials"></div>
     <!-- testimonials: end -->
-    <!-- news: start -->
-    <div id="news"></div>
-    <!-- news: end -->
     <!-- contact: start -->
     <div id="contact"></div>
     <!-- contact: end -->
-    <!-- clients: start -->
-    <div id="clients"></div>
-    <!-- clients: end -->
     <!-- footer: start -->
     <footer id="main-footer"></footer>
     <!-- footer: end -->
+    <!-- section-center-dash: start -->
+    <div id="section-center-dash"></div>
```

Comment:
Manau, kad šitas turėtų būti pridedamas tam tikrų dalių pabaigoje, kaip vaikas (ne visose dalyse jis yra, jei tiksliau, tai paskutinėse trijose dalyse). Šitą reiktų įrašyti atitinkamose užduotyse, kai susirašinėsim užduotis, kiekvienai daliai atskirai.